lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -781,32 +709,6 @@ spec = do
             it ("Quit: " ++ title) $ \ctx ->
                 verifyIt ctx Link.quitStakePool headers expectations
 
-    it "STAKE_POOLS_QUIT_01 - Can quit stake pool" $ \ctx -> do
```

Comment:
removed as it is duplicate of what is in : 
```
it "STAKE_POOLS_JOIN/QUIT - Checking delegation expectations"
```
and we are close to timeout in CI